### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by gelisam**
I had a hard time installing Beluga by following the existing instructions, I figured out how to build it anyway, so I am contributing back the steps which worked for me in the hope it will help the next person.

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -7,7 +7,8 @@ Table of contents:
 1. Installation
   1.1 Generic prerequisites
     1.1.1 Debian/Ubuntu
-    1.1.2 Windows
+    1.1.2 macOS
+    1.1.3 Windows
```
Comment (gelisam) [inline INSTALL]: Before Windows because alphabetical order.

File: INSTALL [hunk 2]
```diff
@@ -24,14 +25,10 @@ Table of contents:
 
 The following packages must be installed to use Beluga:
 
-ocaml 3.12.2+:           http://caml.inria.fr/download.en.html
+ocaml 4.07.1+:           http://caml.inria.fr/download.en.html
```
Comment (gelisam) [inline INSTALL]: One of dependencies of one of the dependencies is a library called "dune" which wouldn't compile with 3.12.2 nor 4.02.0, but did compile with 4.07.1; I can try to find a more accurate minimum version if you prefer.

File: INSTALL [hunk 3]
```diff
@@ -24,14 +25,10 @@ Table of contents:
...
 extlib:                  http://code.google.com/p/ocaml-extlib/downloads/list
 ulex:                    http://www.cduce.org/download.html#side
 
-Users not running Windows may wish to build using omake:
-
-omake:			 http://omake.metaprl.org/index.html
```
Comment (gelisam) [inline INSTALL]: I installed omake but when I ran it, it complained that it couldn't find any omake files; I looked at the history and found that the project is no longer using omake, so this part of the installation instructions was obsolete and misleading.

**Comment by pientka** [comment]
Thanks for taking the time to fix it. 

- Brigitte

**Pull request merged**